src/intrinsics.cpp

```diff
@@ -432,6 +450,19 @@ static jl_cgval_t emit_runtime_call(jl_codectx_t &ctx, JL_I::intrinsic f, const
     return mark_julia_type(ctx, r, true, (jl_value_t*)jl_any_type);
 }
 
+static jl_cgval_t emit_runtime_intrinsic(jl_codectx_t &ctx, jl_cgval_t *f, const jl_cgval_t *argv, size_t nargs)
+{
+    JL_I::intrinsic i = (JL_I::intrinsic)*(uint32_t*)f;
```

Comment:
`f` is a `jl_cgval_t`, not a uint32_t or an intrinsic. We arrived at this call precisely because the value of `f` is not known.

Reply:
We don't know the exact value, but is `f.typ` is `jl_intrinsic_type`, so it should be an intrinsic, right?

Reply:
At runtime, yes, but `f` represents an `llvm::Value` object here